src/main/java/com/cinema/config/validations/PasswordMatchValidator.java

```diff
@@ -0,0 +1,31 @@
+package com.cinema.config.validations;
+
+import com.cinema.model.dto.UserRequestDto;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import org.springframework.beans.BeanWrapperImpl;
+
+public class PasswordMatchValidator
+        implements ConstraintValidator<PasswordMatchConstraint, UserRequestDto> {
+    private String password;
+    private String repeatPassword;
+
+    public void initialize(PasswordMatchConstraint constraint) {
+        this.password = constraint.password();
+        this.repeatPassword = constraint.repeatPassword();
+    }
+
+    public boolean isValid(UserRequestDto userRequestDto, ConstraintValidatorContext context) {
+
+        Object passwordValue = new BeanWrapperImpl(userRequestDto)
```

Comment:
According our qna with Dima should we cast it to String? I have cast it to String, not sure if it is correct. Just double check :)